doc/blackbird_python/example.rst

```diff
@@ -0,0 +1,134 @@
+Example
+=======
+
+This page contains a small guide demonstrating how to use the Python Blackbird parser
+in your Python application, to enable the loading or saving of Blackbird scripts.
+
+
+To start, simply import the Blackbird library into the Python module where it is
+required:
+
+.. code-block:: python
+
+    import blackbird
+
+
+Deserializing Blackbird
+-----------------------
+
+To parse and extract information from a Blackbird text file, with extension ``.xbb``,
+simply use the :func:`~.load` function.
+
+For example, consider the following Blackbird file:
+
+.. code-block:: python
+
+    name StateTeleportation
+    version 1.0
+    target gaussian (shots=1000)
+
+    # state to be teleported:
+    complex alpha = 1+0.5j
+    Coherent(alpha) | 0
+
+    # maximally entangled states
+    Squeezed(-4) | 1
+    Squeezed(4) | 2
+    BSgate(pi/4, 0) | (1, 2)
+
+    # Alice performs the joint measurement
+    # in the maximally entangled basis
+    BSgate(pi/4, 0) | (0, 1)
+    MeasureX | 0
+    MeasureP | 1
+
+    # Bob conditionally displaces his mode
+    # based on Alice's measurement result
+    Xgate(sqrt(2)*q0) | 2
+    Zgate(sqrt(2)*q1) | 2
+
+    MeasureHeterodyne() | 2
+
+
+To import this into our Python module:
+
+.. code-block:: python
+
+    bb = blackbird.load("example/state_teleportation.xbb')

```

Comment:
Github is showing a non-standard character at the end here? Do you see it?

Reply:
Ah, it must be a windows line ending :tired_face: I wrote this file on my windows laptop.

Now that I have linux again, I will quickly replace the line endings before merging